### DISCUSSION TITLE
How to make workspace rule override window rules?

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5318] [Q&A]
I have a bunch of

windowrulev2 = bordersize 0,class:(firefox)

In my config file, but I have a special workspace that I want everything to always have borders in (because there are gaps in the special workspace anyway)

workspace=special,gapsin:20,gapsout:120,bordersize:2,border:true

I've tried the above, but it doesn't effect firefox, how would I achieve this?

**Comment by vaxerski** [comment]
window rules take precedence
